discovery-provider/src/tasks/users.py

```diff
@@ -574,18 +573,6 @@ def recover_user_id_hash(web3, user_id, signature):
     return wallet_address
 
 
-def get_ipfs_metadata(update_task, metadata_multihash, creator_node_endpoint):
-    user_metadata = user_metadata_format
-    if metadata_multihash:
-        user_metadata = update_task.ipfs_client.get_metadata(
-            metadata_multihash,
-            user_metadata_format,
-            creator_node_endpoint,
-        )
-        logger.info(f"index.py | users.py | {user_metadata}")
-    return user_metadata
-
-
```

Comment:
I don't have the most context about this function, but it does look like there is a behavior change with this PR (albeit slight) - previously, we passed in user_metadata_format as the second arg to `get_metadata`, but now it's an empty string. Intentional?

Reply:
Yeah, I did notice this and it seems like this is the only usage of `get_ipfs_metadata()` that we're deprecating.

The only usage is also in `test_index_users.py` so I figured if it were a concern one of the tests would have alerted us. All other business logic has already seemed to move to calling `get_metadata()` directly.

Thoughts?

Reply:
I see the logic - I think the fact that this test didn't fail indicates the test wasn't super broad, rather than this being the right change. Take a look at how we call the ipfs client in index.py:

```            
future = executor.submit(
                update_task.ipfs_client.get_metadata,
                cid,
                track_metadata_format
                if cid_type[cid] == "track"
                else user_metadata_format,
                None,
            )
```
We pass in the user_metadata_format to it, so would be best to replicate that behavior in the test by just passing that same `user_metadata_format` to the raw `ipfs_client.get_metadata` call

Reply:
How does this new commit look?